CalibTracker/SiPixelConnectivity/test/SiPixelFedCablingMapWriter_cfg.py

```diff
@@ -2,9 +2,10 @@
 
 process = cms.Process("MapWriter")
 process.load("CondCore.DBCommon.CondDBCommon_cfi")
-process.CondDBCommon.connect = cms.string("sqlite_file:cabling.db")
 
-process.load("Configuration.StandardSequences.Geometry_cff")
+#process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
+
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff")
```

Comment:
this can be moved safely back to `Configuration.StandardSequences.FrontierConditions_GlobalTag_cff"`